examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
-
-echo "Cleaning up openshift runtime files ..."
-rm -rf openshift.local.*
+sudo pkill -x openshift
 
 echo "Stopping all k8s docker containers on host ..."
 docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
+
+echo "Unmounting openshift local volumes ..."
+mount | grep "openshift.local.volumes" | awk '{ print $3}' | xargs -l -r sudo umount
+
+echo "Cleaning up openshift runtime files ..."
+sudo rm -rf openshift.local.*
```

Comment:
pretty sure it's necessary, it includes dirs that were created by openshift running as root.